<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutDialog</class>
 <widget class="QDialog" name="AboutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>228</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>About Joker</string>
  </property>
  <widget class="QLabel" name="labelCpRight">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>351</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Copyright 2013-2014 Phonations, All rights reserved.</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelVersion">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">Joker vX.X.X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsViewIcon">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="backgroundBrush">
    <brush brushstyle="NoBrush">
     <color alpha="0">
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
    </brush>
   </property>
   <property name="foregroundBrush">
    <brush brushstyle="NoBrush">
     <color alpha="0">
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
    </brush>
   </property>
  </widget>
  <widget class="QLabel" name="labelContact">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">support@phonations.com</string>
   </property>
   <property name="openExternalLinks">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
